engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/RuntimeServerTest.scala

```diff
@@ -1163,6 +1163,10 @@ class RuntimeServerTest
     val moduleName = "Test.Main"
     val metadata   = new Metadata
 
+    @scala.annotation.unused
+    val fooDefinition = metadata.addItem(26, 22)
+    @scala.annotation.unused
+    val fooName = metadata.addItem(26, 3)
```

Comment:
Why assign these if they're not used? Just call `metadata.addItem`.

Reply:
The name is just a note to keep track of what is annotated. I can use a comment instead :+1: